test/helpers/pkcs12.c

```diff
@@ -90,9 +90,7 @@ static EVP_PKEY *load_pkey_asn1(const unsigned char *bytes, int len)
     EVP_PKEY *pkey = NULL;
 
     pkey = d2i_AutoPrivateKey(NULL, &bytes, len);
-    if (!TEST_ptr(pkey))
-        goto err;
-err:
```

Comment:
I'm not entirely sure why this one is being changed but not the one above at line 83.

But more generally: this is part of a pattern that makes it easier to extend test cases in the future if we need to.

What is the specific problem with keeping a redundant `goto` around?